### DISCUSSION TITLE
How to generate c++/cx project to combine with unity project on the windows platform ?

### DISCUSSION THREAD

**Discussion by thienphuoc** [microsoft/react-native-windows#10334] [Q&A] [answered]
Hello everybody,

I combine successfully the react-native with the unity project on android/ios platforms.

Now, I want to combine the unity project with the react-native-windows project on the windows platform. The react-native-project only generates to C++/Winrt project ( I ignore c# because unity cannot generate to this type), right? The unity engine (2020->2022) only generates to C++/CX project.

I'm not family with C++/Winrt, and I cannot combine two projects with different types? So now, Could everyone help me solve this issue ? Can I generate a C++/CX project from react-native-windows or give me an example regarding react-native-windows and unity integration?

Ex: npx react-native-windows --override --language cppcx

**Comment by chrisglein** [comment, marked as answer]
RNW doesn't support C++/CX. But you should be able to create a C# Unity app. That'd be the recommended path for you.
